server/@types/welcome/index.d.ts

```diff
@@ -5,9 +5,9 @@ declare module 'welcome' {
   export type NewOffenderBooking = schemas['NewOffenderBooking']
   export type ImprisonmentStatus = schemas['ImprisonmentStatus']
   export type Prison = schemas['Prison']
-  export type OffenderNumber = schemas['OffenderNumber']
   export type PrisonNumber = schemas['PrisonNumber']
```

Comment:
is PrisonNumber no long being used?

Reply:
No - think that's what we wanted though for the frontend?